packages/examples/project-b/federation.config.js

```diff
@@ -5,9 +5,7 @@ export const federationconfig = {
     './button': './src/index.js',
   },
   shared: {
-    react: {
-      import: false,
-    },
-    axios: {}
```

Comment:
This is done because @module-federation/runtime doesn't support `import: false` yet.

Bug: https://github.com/module-federation/universe/issues/2020